gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -283,15 +311,64 @@ func (vm *VMKeeper) Call(ctx sdk.Context, msg MsgCall) (res string, err error) {
 		}
 		m.Release()
 	}()
+
 	rtvs := m.Eval(xn)
-	ctx.Logger().Info("CPUCYCLES call", "num-cycles", m.Cycles)
-	for i, rtv := range rtvs {
-		res = res + rtv.String()
-		if i < len(rtvs)-1 {
-			res += "\n"
+
+	// Result
+
+	// Iterate through params
+	var response gno.TypedValue
+	{
+		var sv gno.StructValue
+		var st gno.StructType
+
```

Comment:
Nitpick: you can combine this under a single `var` group

Reply:
can depend on the context, but when i've no more that 2 vars, i feel it's more clear to keep them with two separate declaration and not having the nested indent + less lines